specification/machinelearningservices/resource-manager/readme.go.md

```diff
@@ -13,9 +13,29 @@ go:
 
 ```yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2019-05-01
+  - tag: package-2018-11-19
   - tag: package-2018-03-preview
 ```
 
+### Tag: package-2019-05-01 and go
+
+These settings apply only when `--tag=package-2019-05-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag)=='package-2018-03-preview' && $(go)
+output-folder: $(go-sdk-folder)/services/machinelearning/mgmt/2019-05-01/$(namespace)
+```
+
+### Tag: package-2018-11-19 and go
+
+These settings apply only when `--tag=package-2018-11-19 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag)=='package-2018-03-preview' && $(go)
```

Comment:
The tag is wrong, it should be `package-2018-11-19`

Reply:
Great Thanks! The issue fixed now and it is the root cause!!